scripts/sysman/firmware.yml

```diff
@@ -90,6 +90,49 @@ returns:
     - $X_RESULT_ERROR_INSUFFICIENT_PERMISSIONS:
         - "User does not have permissions to perform this operation."
 --- #--------------------------------------------------------------------------
+type: function
+desc: "Find Firmware Flash Progress"
+version: "1.7"
+class: $sFirmware
+name: UpdateProgress
```

Comment:
this is a getter or a setter? I think it is a getter, so, could you add the verb explicitly? like GetProgress ? this would answer also the question from @aravindksg below, if this requires sudo (usually getters dont need sudo).

Reply:
Agree - this is a getter. Make sense to change to GetProgress

Reply:
If possible, we can have the API specific to Flash progress, something like zesFirmwareGetFlashProgress

Reply:
API name changed